main_server.py

```
@@ -2041,6 +3048,935 @@ def search_files_recursive(directory, target_ext, result_list):
+                }
+            }
+        ]
+        
+        return JSONResponse(content={"success": True, "items": items})
+        
+    except Exception as e:
+        logger.error(f"获取订阅的创意工坊物品失败: {e}")
+        return JSONResponse(content={"success": False, "error": str(e)}, status_code=500)
+
+@app.get('/api/file-exists')
+async def check_file_exists(path: str = None):
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)